management/nodes/startupScripts/win2016-201700802-internal.ps1

```diff
@@ -0,0 +1,64 @@
+Set-ExecutionPolicy Unrestricted
+
+$machineSetupLog = 'C:\setup.log'
+$jenkinsServerUrl = $args[0]
```

Comment:
>$args[0] [](start = 20, length = 8)

These parameters should probably be listed in a `param()` block at the top of the file, rather than manually extracted here.